README.md

```diff
@@ -89,6 +89,7 @@ your browser.
 ### macOS
 
 [![macOS](https://img.shields.io/badge/macOS-Docker-000000?style=for-the-badge&logo=macos&logoColor=F0F0F0)](https://memgraph.com/docs/memgraph/install-memgraph-on-macos-docker)
+[![macOS](https://img.shields.io/badge/lima-AACF41?style=for-the-badge&logo=macos&logoColor=F0F0F0)](https://memgraph.com/docs/memgraph/install-memgraph-db-on-macos-docker)
```

Comment:
We need a special page under docs for Lima but in the absence of that page, I think the most relevant page to link is https://memgraph.com/docs/memgraph/install-memgraph-on-ubuntu

Reply:
I've updated the link.